kythe/typescript/indexer.ts

```diff
@@ -1549,12 +1565,31 @@ class Visitor {
    *     ((local var)|const|let) <name>: <type>( = <initializer>)?
    * where `(local var)` is the declaration of a variable in a catch clause.
```

Comment:
Does this need an update for the new binding support?

Reply:
No because the form of the fact is the same. We don't emit information other than this